src/numth/misc.cpp

```diff
@@ -1,13 +1,15 @@
+/**
+ * Author: Richard Hladik
+ * Description: Returns $a^p\ \%\ mod$.
```

Comment:
"%" for "modulo" in math mode doesn't look very good to me. I think \bmod instead of \% (and maybe enclosing "mod" in {\tt}) would be more appropriate here.